Sources/Swift/Persistence/SentryScopeContextPersistentStore.swift

```diff
@@ -0,0 +1,83 @@
+@_implementationOnly import _SentryPrivate
+
+@objcMembers
+class SentryScopeContextPersistentStore: NSObject {
```

Comment:
`m`: Codecov complains rightfully about the test coverage of this class. Can we maybe add some unit tests, covering the edge cases?